src/i18n/en/reactApp/deployments.reactApp.i18n.json

```diff
@@ -14,13 +14,25 @@
 		"error": "Error - please reload"
 	},
 	"noDeployments": "No deployments found",
-	"executeEntrypoint": "Entrypoint",
+	"executeFunctionName": "Entrypoint function",
 	"executionFunctionsNotFound": "No functions found",
 	"executeFile": "File",
+	"errors": {
+		"missingFile": "Please choose a file",
+		"missingFunction": "Please choose a function"
```

Comment:
```suggestion
		"missingFunction": "Please write a function"
```

Reply:
refactored to: "Please type a function name"